spring-cloud-contract-stub-runner/src/main/java/org/springframework/cloud/contract/stubrunner/provider/wiremock/WireMockHttpServerStub.java

```diff
@@ -237,15 +238,25 @@ private String jsonArrayOfMappings(Collection<String> mappings) {
 
 	@Override
 	public boolean isAccepted(File file) {
-		return file.getName().endsWith(".json");
+		return file.getName().endsWith(".json") && validMapping(file);
```

Comment:
Cool! Can we add a test for that? Or do we have one already?

Reply:
Will add some tests.

Reply:
Added tests.